app/src/main/java/com/github/swent/echo/compose/event/EventScreen.kt

```diff
@@ -150,7 +154,8 @@ fun EventPropertiesFields(eventViewModel: EventViewModel) {
         }
         EventDropDownSelectOrganizer(
             organizerName = organizerName,
-            organizerList = organizerListState.value
+            organizerList = organizerListState.value,
+            enabled = isOnline
         ) {
             eventViewModel.setOrganizer(it)
         }
```

Comment:
It might be worth it to pass the `isOnline` parameter to the `EventLocationEntry` so that the map can be hidden, or at least so that the location edition can be disabled.

Reply:
Thanks for the suggestion